### PR TITLE
Add wormhole and update the list

### PR DESCRIPTION
**Pull request by vimode**
* [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

## New additions 
-  [wormhole.app](https://wormhole.app/) a file sharing service with E2E encryption.
- Brevo replaces SendinBlue
 
## Updates
 
 - [Sendpulse](https://sendpulse.com/prices)

- [APIMocha](https://apimocha.com/app/faq)

## Removals

- IPloka doesn't seem to be online anymore.

- Validator.Pizza - Redirects to [Mailcheck.ai](https://www.mailcheck.ai) (already in the list)

- SendinBlue - Redirects to [Brevo](https://www.brevo.com/).  So edited to be added as Brevo in the list.

- skiff  - Now acquired by Notion and will be [sunsetting its offering in August 2024](https://skiff.com/data-migration).

- [IPTrace](iptrace.io) - Now  part of ipstack (already in the list)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [IP.City](https://ip.city) — 100 Free IP geolocation requests per day
   * [Abstract API](https://www.abstractapi.com) — API suite for various use cases, including IP geolocation, gender detection, or email validation.
   * [Apify](https://www.apify.com/) — Web scraping and automation platform to create an API for any website and extract data. Ready-made scrapers, integrated proxies, and custom solutions. Free plan with $5 platform credits included every month.
-  * [API Mocha](https://apimocha.com) - Completely free online API mocking for testing and prototyping.  Make up to 500 daily requests, fully customizable API responses, and download mock rules as a Postman collection.
+  * [API Mocha](https://apimocha.com) - Completely free online API mocking for testing and prototyping. Make up to 50 API requests in a rolling 24 hour period, fully customizable API responses, and download mock rules as a Postman collection.
```
Comment (ripienaar) [inline README.md:222]: hmm, I think this is just not useful anymore with such a low limit.  We can remove it

File: README.md [hunk 2]
```diff
@@ -1900,7 +1898,7 @@ Update Time, five active automations, webhooks.
   * [Buff.tools](https://buff.tools/) - An all-in-one digital toolbox featuring Web, SEO, AI, Domain Management, unit conversion, calculators, and Image Manipulation Tools.
   * [ElevateAI](https://www.elevateai.com) - Get up to 200 hours of audio transcription for free every month.
   * [get.localhost.direct](https://get.localhost.direct) — A better `*.localhost.direct` Wildcard public CA signed SSL cert for localhost development with sub-domain support
-  * [Framacloud](https://degooglisons-internet.org/en/) — A list of Free/Libre Open Source Software and SaaS by the French non-profit [Framasoft](https://framasoft.org/en/).
+  * [Framacloud](https://degooglisons-internet.org/en/list/) — A list of Free/Libre Open Source Software and SaaS by the French non-profit [Framasoft](https://framasoft.org/en/).
```
Comment (ripienaar) [inline README.md:1901]: this is reverting a PR I just merged :)

**Review by ripienaar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vimode) [reply README.md:1901]: :D Oh! Didn't realise that happened. Let me fix it.

**Comment by ripienaar** [comment]
nice, thanks

**Pull request merged**